apps/server/src/modules/federal-state/service/federal-state.service.ts

```diff
@@ -16,4 +17,14 @@ export class FederalStateService {
 		const federalStates: FederalStateDO[] = await this.federalStateRepo.findAll();
 		return federalStates;
 	}
+
+	async create(federalStateCreate: IFederalStateCreate) {
```

Comment:
add return type for better readability
somehow is not ok to pass through the params basically from controller all the way to the repo
we need a mapping layer in between... 
you can add a domain object factory class, to instanciate the DO object before passing it to the repo